### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).

It also includes following components:

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties [hunk 1]
```diff
@@ -192,3 +192,10 @@ recovery-codes-generate-new-codes=Generieren Sie neue Wiederherstellungscodes, u
 error-invalid-multivalued-size=Attribut {0} muss mindestens {1} und höchstens {2} Wert(e) haben.
 recovery-authn-code=Meine Authentifizierungscodes für die Wiederherstellung
 recovery-authn-codes-help-text=Diese Codes können verwendet werden, um Ihren Zugang wiederherzustellen, falls Ihre anderen Zwei-Faktor-Mittel nicht verfügbar sind.
+verifiableCredentialsIssuerAlert=War nicht in der Lage, die Ausstellungsinformationen abzurufen.
+verifiableCredentialsConfigAlert=War nicht in der Lage, die Berechtigungsnachweiskonfiguration abzurufen.
+oid4vci=Überprüfbare Berechtigungsnachweis
+verifiableCredentialsTitle=Überprüfbare Berechtigungsnachweis
```
Comment (ahus1) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties:198]: I think this should be at least a plural ("Berechtigungsnachweise"). 

@thomasdarimont - as one who is involved with Keycloak's OpenID Connect SIG and a German native speaker: Could you please comment how this should be translated, and if it should be translated at all?

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for updating the German langage, @robson90!

See below for one change I'd suggest. I'd also like to get some feedback on those who are involved in this effort to make Keycloak compatible with that standard of verifiable credentials.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties (hunk 1 above)
Comment (thomasdarimont) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties:198]: I agree with you @ahus1 that plural `Verifizierbare Berechtigungsnachweise` seems to be correct.
See: https://www.selfsovereignidentity.it/einfuhrung-in-verifiable-credentials/

Comment (robson90) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties:198]: @ahus1 & @thomasdarimont  thanks for the clarification. Changes are incoming.

**Review by ahus1 — APPROVED** [review]

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties (hunk 1 above)
Comment (ahus1) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties]: Thank you for this translation!

**Pull request merged**